frontend/src/metabase/admin/upsells/components/Upsells.styled.tsx

```diff
@@ -55,9 +56,23 @@ export const UpsellCTALink = styled(ExternalLink)`
   }
 `;
 
-export const UpsellCardComponent = styled.div`
-  max-width: 200px;
-  box-sizing: content-box;
```

Comment:
[For reference](https://github.com/metabase/metabase/pull/45424#discussion_r1674093953), box-sizing is removed so it will be `box-sizing: border-box` by default

https://github.com/metabase/metabase/blob/7e6adfaa3f3290601162135f546cdcaa3c120a85/frontend/src/metabase/css/core/box_sizing.module.css#L1-L19